website/examples/custom-selection-single.tsx

```diff
@@ -1,19 +1,18 @@
 import React from "react";
-import { DayClickEventHandler, DayPicker } from "react-day-picker";
+import { DayPicker } from "react-day-picker";
 
 export function Example() {
-  const [selectedDay, setSelectedDay] = React.useState<Date>();
-  const handleDayClick: DayClickEventHandler = (day) =>
-    setSelectedDay(day);
+  const [selected, setSelected] = React.useState<Date>();
 
-  const footer = selectedDay
-    ? `You selected ${selectedDay.toLocaleDateString()}.`
-    : "Please pick a day.";
+  let footer = "Please pick a day.";
+  if (selected) {
+    footer = `You picked ${selected.toLocaleDateString()}.`;
+  }
   return (
     <DayPicker
-      mode="custom"
-      selected={selectedDay}
-      onDayClick={handleDayClick}
```

Comment:
We don’t need onDayClick anymore? I’d like to test this a little bit. I’m not looking at the code right now, but I’m unfamiliar with the code calls onSelect in the custom mode

Reply:
Sorry for the confusion, that was a wrong copy/paste. The example is restored in dd748c5851f53e52d20326c7cb4d9c874c934cf1.